locale/en/manageiq.po

```diff
@@ -18184,15 +18217,15 @@ msgstr ""
 msgid "Edit Tags for the selected Ansible Repositories"
 msgstr ""
 
-msgid "Edit Tags for the selected Ansible Tower Providers"
-msgstr ""
-
```

Comment:
@agrare @GilbertCherrie can you verify the ansible tower strings that remain?  For example, not sure if ansible tower should be in these more generically named automation manager files: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar/ems_automation_center.rb#L53 or https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar/ems_automation_center.rb#L33-L34